torchtext/data/utils.py

```diff
@@ -71,7 +71,7 @@ def _basic_english_normalize(line):
     return line.split()
 
 
-def get_tokenizer(tokenizer, language='en'):
+def get_tokenizer(tokenizer, language='en', spm_name=None):
```

Comment:
This flag seems unrelated and too general for a geneic "get_tokenizer" function. If spm_name is not None for a tokenizer that's not "sentencepiece" it has no effect or meaning.